src/ocrd_network/runtime_data/network_agents.py

```diff
@@ -0,0 +1,131 @@
+from logging import Logger
+from typing import Any
+
+from re import search as re_search
+from ..constants import AgentType, DeployType
+
+
+# TODO: Find appropriate replacement for the hack
+def deploy_agent_native_get_pid_hack(logger: Logger, ssh_client, start_cmd: str):
+    channel = ssh_client.invoke_shell()
+    stdin, stdout = channel.makefile("wb"), channel.makefile("rb")
+    logger.debug(f"Executing command: {start_cmd}")
+
+    # TODO: This hack should still be fixed
```

Comment:
Do we have an issue tracking this?

Reply:
Not sure - maybe not. I am planning to rework this after the module tests where it will be easier for me to try different things and just run module tests to verify different solutions.